extensions/typescript-language-features/src/typescriptServiceClient.ts

```diff
@@ -1038,15 +1039,20 @@ export default class TypeScriptServiceClient extends Disposable implements IType
 				return;
 
 			case EventName.createDirectoryWatcher: {
-				const path = (event.body as Proto.CreateDirectoryWatcherEventBody).path;
-				if (path.startsWith(inMemoryResourcePrefix)) {
+				const fpath = (event.body as Proto.CreateDirectoryWatcherEventBody).path;
```

Comment:
Wondering if we need similar code for the `case EventName.createFileWatcher:` below? Is it possible that in memory files are watched by accident?

Reply:
Similar code exists already, as of https://github.com/microsoft/vscode/commit/fd5f15a2b33d1bbce7aa7bca80a98ae37e6ac81d

The diff here is confusing because I had to rename the `path` variable to be able to use the `path.join` package method

Reply:
Sorry, good point!